build/ci/.azure-devops-package-net6-win.yml

```
@@ -97,6 +97,8 @@ jobs:
 
   - task: PublishBuildArtifacts@1
     condition: always()
+    retryCountOnTaskFailure: 3
+    retryCountOnTaskFailure: 3
```

looks incorrectly duplicated?

```suggestion
    retryCountOnTaskFailure: 3
```